### PR TITLE
move configuration to config directory so it can be mounted in docker

### PR DESCRIPTION
**Pull request by zapthedingbat**

### CODE REVIEW

**Comment by zapthedingbat** [comment]
I've been able to build this with 
```
docker build -t sim-login .
```

then run it with 
```
docker run -v ${pwd}/config:/app/config -p 3000:8080 -it --rm --init sim-login
```
to mount the config files

**Comment by pauldougan** [comment]
WillTake a look tomorrow morning.

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,7 @@
 /node_modules/
 /dist/
-/.keys/
+/public/
```
Comment (zapthedingbat) [inline .gitignore:3]: This is just a bit of a clean up - /public/ is generated at build time so no need to commit it to version control

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,7 @@
 /node_modules/
 /dist/
-/.keys/
```
Comment (zapthedingbat) [inline .gitignore:3]: /.keys/ directory isn't used. Although there's an unrelated issue to persist the spot key that gets generated in a docker volume

File: .gitignore [hunk 3]
```diff
@@ -1,9 +1,7 @@
 /node_modules/
 /dist/
-/.keys/
+/public/
 .env
 *.pem
 *.jwk
-/automation/TestData.ts
-/automation/.screenshots/
```
Comment (zapthedingbat) [inline .gitignore:8]: Cleaning this up - there's no automation directory

File: Dockerfile [hunk 4]
```diff
@@ -30,11 +30,16 @@ WORKDIR /app
...
-COPY --from=build-env /app/public ./public
-COPY ./config ./config
+COPY --from=build-env /app/dist /app/bin
+COPY --from=build-env /app/public /app/public
+COPY --chown=app --chmod=777 ./config /app/config
+
+VOLUME /app/config/keys
 
 ENV NODE_PORT=${NODE_PORT}
+ENV USER_CONFIG_DIR="/app/config/users"
```
Comment (zapthedingbat) [inline Dockerfile:40]: Configure where to load users and clients configuration, in case we want to mount it elsewhere in the container

**Pull request merged**